reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -652,6 +653,26 @@ private fun DIV.licenseLink(license: String) {
     }
 }
 
+private fun DD.licensesLink(expression: SpdxExpression) {
+    val licenses = expression.decompose().map { it.toString() }
+
+    val licenseComponents = "$expression".split(" ")
+    div {
+        val licenseIterator = licenseComponents.iterator()
+        while (licenseIterator.hasNext()) {
+            val license = licenseIterator.next()
+            if (licenses.contains(license)) {
```

Comment:
This could also be an operator, as explained [here](https://github.com/oss-review-toolkit/ort/pull/3830#discussion_r604758863) But i see how the variable name "license" is missleading